src/components/bcros/filing/addStaffFiling/Index.vue

```diff
@@ -81,7 +81,7 @@ const allActions: ComputedRef<Array<MenuActionItem>> = computed(() => {
       datacy: 'court-order',
       label: t('label.filing.staffFilingOptions.courtOrder'),
       click: () => {
-        goToFilingsUI(`/${currentBusiness.value.identifier}/court-order'`, { filingId: '0' })
+        goToFilingsUI(`/${currentBusiness.value.identifier}/court-order`, { filingId: '0' })
```

Comment:
Does this work now?

Can you fix the others above/below with the same extra character, and test them?